static/src/javascripts/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -118,27 +118,28 @@ const renderQuickSurvey = () => {
     // look for a survey with this ID to return. This is useful as we can easily see how a particular survey
     // would be rendered, without actually putting it live. If this parameter is empty or not specified, tailor
     // behaves as usual.
-    const surveyToShow = local.get('surveyToShow');
+    const surveyToShow = localStorage.surveyToShow || [];
 
     if (surveyToShow) {
-        queryParams.surveyToShow = surveyToShow;
+        queryParams.force = surveyToShow;
     }
 
     // get the list of surveys that can't be shown as they have been shown recently
     const surveysNotToShow = getSurveyIdsNotToShow();
 
     if (surveysNotToShow) {
```

Comment:
If I'm reading this correctly, `surveysNotToShow` will always be truthy, so perhaps this check is redundant?